examples/README.md

```diff
@@ -9,25 +9,111 @@ This collection of sample templates will help you get started with MongoDB Atlas
+You should create a profile in the AWS Secrets Manager that contains the MongoDB Atlas Programmatic API Key.
 
-**Option 2: Use the provided CFN template to create an IAM execution role and activate extensions**
+Use [this template](profile-secret.yaml) to create a [new CloudFormation stack](https://console.aws.amazon.com/cloudformation/home#/stacks/create) for the default profile that all resources attempt to use unless you specify a different profile.
 
-Use the [CFN template provided](activate-mongodb-atlas-resources.template.yaml) to create a new stack in required AWS region which will activate all required MongoDB Atlas extensions in your account in the specified region.
+### Configure the IAM Execution Role
+CloudFormation extensions use an *execution role* ([IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)) to access AWS resources.
+You must configure this role must with at least a policy allowing `secretsmanager:GetSecretValue` to access the configured [profile](../README.md#2-configure-your-profile).
 
-_Note:_
-_If the stack is created to activate already activated extensions in the region, you will run into failures as AWS does not allow to use `TypeActivation` to re-activate extensions._
+To create the execution role, do one of the following steps:
+- Use the [execution-role.yaml](execution-role.yaml) template
+- Create the execution-role as part of the [next step](#activate-the-mongodb-atlas-cloudformation-public-extensions)
+- Use the [AWS Console](https://us-east-1.console.aws.amazon.com/iam/home?region=us-east-1#/roles) or another method of your choice.
 
+### Activate the MongoDB Atlas CloudFormation public extensions
+You have the following options for activating the extensions, for example `MongoDB::Atlas::Cluster`:
 
-### CloudFormation Profile
-A profile should be created in the AWS Secrets Manager, containing the MongoDB Atlas Programmatic API Key.
+- Use the [Cloudformation service in the AWS Console](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/registry/public-extensions?visibility=PUBLIC&type=RESOURCE&category=THIRD_PARTY):
+   1. Ensure you are in the correct AWS region.
+   2. Select Publisher=`ThirdParty`.
+   3. Extension name prefix = `MongoDB::Atlas`.
+   4. Select the resource type and click `Activate`.
+   5. In the *Execution role ARN* specify the role arn, for example `arn:aws:iam::123456789012:role/cfn-execution-role`, from the previous step.
+- Use the provided [CFN template](activate-mongodb-atlas-resources.template.yaml) to create an IAM execution role and activate all MongoDB Atlas extensions:
+   1. Specify the Region.
+   2. [Create the stack](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks?filteringText=&filteringStatus=active&viewNested=true). If the stack has been run before in the same region, you will run into failures as AWS does not allow to use `TypeActivation` to re-activate extensions.
```

Comment:
Do we have any guidance to what users can do in this case? Or is this a AWS CFN limitation and they are just out of luck? If the later would suggest have a call out in here to advise users to create a support ticket with AWS directly for assistance.

Reply:
@Zuhairahmed thanks for the feedback 😁
Does this commit 32f8943928da70c3a5ac4028852e477abd8106f3 help sufficiently?